src/Persisters/Collection/OneToManyPersister.php

```diff
@@ -166,7 +168,10 @@ public function loadCriteria(PersistentCollection $collection, Criteria $criteri
         throw new BadMethodCallException('Filtering a collection by Criteria is not supported by this CollectionPersister.');
     }
 
-    /** @throws DBALException */
+    /**
+     * @throws EntityNotFoundException
+     * @throws MappingException
+     */
```

Comment:
This replaces DBALException with ORMExceptions. Is this an additional fix because none of them extends the Exception of DBAL?

Reply:
DBALException should not have been removed - my IDE autofix appears to have removed that line by mistake. I have pushed an amendment to fix that.

The ORM exceptions are potentially thrown by `UnitOfWork::getEntityIdentifier()` (EntityNotFoundException) and `ClassMetadataInfo::getFieldForColumn()` (MappingException).

Reply:
I'm not sure if this is a breaking change as this is going to be merged up to 3 and 4 of ORM. But it looks like the DBALException isn't handled either by ORM despite being an Exception of a different library. Now ORMExceptions were added too.